parent/pom.xml

```diff
@@ -0,0 +1,516 @@
+
+        <javac.target>${javac.source}</javac.target>
+        <project.build.java.target>${javac.target}</project.build.java.target>
+        <maven.compiler.target>${javac.target}</maven.compiler.target>
+        <ant.build.javac.target>${javac.target}</ant.build.javac.target>
+
+        <min.required.maven.version>3.2</min.required.maven.version>
+        <surefire.plugin.version>2.19</surefire.plugin.version>
+            <groovy.major.version>2</groovy.major.version>
+            <groovy.minor.version>4</groovy.minor.version>
+            <groovy.release.version>5</groovy.release.version>
+            <groovy.compliance.level>2.0</groovy.compliance.level>
+        <groovy.version>${groovy.major.version}.${groovy.minor.version}.${groovy.release.version}</groovy.version>
+        <gmaven.plugin.version>1.5</gmaven.plugin.version>
+
+        <ant.version>1.9.6</ant.version>
+        <xmltask.version>1.16</xmltask.version>       
+        <ant-contrib.version>1.0b3</ant-contrib.version>
+        <junit.version>4.12</junit.version>
+    </properties>
+
+    <prerequisites>
+        <maven>${min.required.maven.version}</maven>
+    </prerequisites>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.apache.ant</groupId>
+                <artifactId>ant</artifactId>
+                <version>${ant.version}</version>
+                <exclusions>    <!-- provided by the JDK -->
+                    <exclusion>
+                        <groupId>xerces</groupId>
+                        <artifactId>xercesImpl</artifactId>
+                    </exclusion>
+                    <exclusion>
+                          <groupId>xml-apis</groupId>
+                          <artifactId>xml-apis</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.ant</groupId>
+                <artifactId>ant-nodeps</artifactId>
+                <version>${ant.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.ant</groupId>
+                <artifactId>ant-launcher</artifactId>
+                <version>${ant.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.ant</groupId>
+                <artifactId>ant-commons-logging</artifactId>
+                <version>${ant.version}</version>
+                <exclusions>    <!-- we use jcl-over-slf4j -->
+                    <exclusion>
+                          <groupId>commons-logging</groupId>
+                          <artifactId>commons-logging-api</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>${junit.version}</version>
```

Comment:
I don't think we use junit yet, so one could argue it shouldn't be added until we do. 
If you decide to keep it in, I think it should be declared with `<scope>test</scope>`